src/components/TileSelect/TileCollapser.js

```diff
@@ -121,19 +129,10 @@ const SelectCollapser = ({
             </Link>
           )}
         </h5>
-        <Icon
+        <AnimatedFeatherIcon
+          style={style}
           name="fe-chevron-right"
           size="1rem"
```

Comment:
We can remove this since its a default prop yes?

Reply:
@caylahamann looks like 1em is the default, any reason we're using em and rem for this?

Reply:
@LizBaker no i literally just copy pasta'd from the SVG component in gatsby-theme-newrelic 
https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/SVG.js#L27-L29
so i'm really not sure